### PR TITLE
Mp links implementation

### PR DESCRIPTION
**Pull request by lenforiee**
This PR needs to be merged as first

### CODE REVIEW

File: app/handlers/multiplayer/multiplayer.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		if err != nil {
+			c.Error(err)
+			slog.ErrorContext(c, err.Error())
+		}
+
+		err = services.DB.Get(
+			&participantsIds,
+			`SELECT JSON_ARRAYAGG(user_id) FROM (
+				SELECT DISTINCT(user_id) AS user_id FROM 
+				match_events WHERE match_id = ? AND event_type = 'MATCH_USER_JOIN'
```
Comment (tsunyoku) [inline app/handlers/multiplayer/multiplayer.go]: maybe move `MATCH_USER_JOIN` into a constant

File: app/handlers/multiplayer/multiplayer.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		)
+
+		if err != nil {
+			c.Error(err)
+			slog.ErrorContext(c, err.Error())
+		}
+	}
+
+	data.BannerContent = "/static/images/headers/2fa.jpg"
+	data.BannerAbsolute = true
```
Comment (tsunyoku) [inline app/handlers/multiplayer/multiplayer.go]: `data.BannerContent = "2fa.jpg"` without setting `BannerAbsolute` should work here

**Review by tsunyoku — CHANGES_REQUESTED** [review]

File: app/handlers/multiplayer/multiplayer.go (hunk 2 above)
Comment (lenforiee) [reply app/handlers/multiplayer/multiplayer.go]: resolved in 113fa86fb4374005ed7838712bf76b31068fb6d2

File: app/handlers/multiplayer/multiplayer.go (hunk 1 above)
Comment (lenforiee) [reply app/handlers/multiplayer/multiplayer.go]: resolved in ac6a703ecf09150036c95c7c7976240346f97a88

**Review by tsunyoku — APPROVED** [review]

**Pull request merged**